pkg/controller/controller_utils.go

```diff
@@ -962,6 +961,10 @@ func IsPodActive(p *v1.Pod) bool {
 		p.DeletionTimestamp == nil
 }
 
+func IsPodTerminating(p *v1.Pod) bool {
```

Comment:
I'm hesitant if we need this as a function, since in many places in code we already check the DeletionTimestamp directly, and use it in KEPs as a direct way to check if a pod is terminating. It sort of makes sense to have a function that expresses the intention by its name, but it does not seem needed for this PR (as we already have it in `gc_controller`, so I would suggest reverting it.

Reply:
Removed. PTAL.